package com.credible.amfi.response;


public class ApiCalculatorLumpsum {
	
	public Integer status;
	public String status_msg;
	public String msg;
    private long lumpsum_amount;
    private Double expected_return;
    private Double years;
    private long future_amount;
    
	public Integer getStatus() {
		return status;
	}
	public void setStatus(Integer status) {
		this.status = status;
	}
	public String getStatus_msg() {
		return status_msg;
	}
	public void setStatus_msg(String status_msg) {
		this.status_msg = status_msg;
	}
	public String getMsg() {
		return msg;
	}
	public void setMsg(String msg) {
		this.msg = msg;
	}
	public long getLumpsum_amount() {
		return lumpsum_amount;
	}
	public void setLumpsum_amount(long lumpsum_amount) {
		this.lumpsum_amount = lumpsum_amount;
	}
	public Double getExpected_return() {
		return expected_return;
	}
	public void setExpected_return(Double expected_return) {
		this.expected_return = expected_return;
	}
	public Double getYears() {
		return years;
	}
	public void setYears(Double years) {
		this.years = years;
	}
	public long getFuture_amount() {
		return future_amount;
	}
	public void setFuture_amount(long future_amount) {
		this.future_amount = future_amount;
	}
}
